src/components/Card.tsx

```diff
@@ -0,0 +1,128 @@
+import {
+  SwipeableListItem,
+  Type,
+  SwipeAction,
+  TrailingActions,
+} from "react-swipeable-list";
+import { format, isToday } from "date-fns";
+import { de } from "date-fns/locale";
+import clsx from "clsx";
+import { Transition } from "@headlessui/react";
+import { useState } from "react";
+import "react-swipeable-list/dist/styles.css";
+import { Trash, Bookmark } from "react-feather";
+
+type Props = {
+  data: {
+    title?: string;
+    dateCreated: Date;
+    checkedItems: number;
+    totalItems: number;
+  };
+  createNewCard?: Boolean;
+  favorite?: Boolean;
+  onRemove: Function;
+};
+
+export default function Card({
+  data: { dateCreated, checkedItems, totalItems, title },
+  createNewCard,
+  favorite,
+  onRemove,
+}: Props) {
+  const [swiped, setSwiped] = useState(true);
+  const [inDeletion, setIndDeletion] = useState(false);
+  const [cardBackgroundSwipe, setCardBackgroundSwipe] = useState("");
+
+  function deleteListItem() {
+    setIndDeletion(true);
+    setSwiped(false);
+    onRemove();
+  }
+
+  const trailingActions = () => (
+    <TrailingActions>
+      <SwipeAction onClick={() => deleteListItem()}>
+        <DeleteList />
+      </SwipeAction>
+    </TrailingActions>
+  );
+
+  function styleSwipe(direction: string) {
+    setCardBackgroundSwipe(direction);
+  }
+
+  const basicCardStyle = "rounded-2xl  w-full h-44 ";
+
+  return (
+    <Transition
+      show={swiped}
+      appear={true}
+      enter="transform transition duration-[400ms]"
```

Comment:
I've seen `duration-[400ms]` often in the project, maybe it's time for an entry in the tailwind config?

Reply:
added in 1c1c16d